client/new-stack-editor/lib/views/flexsplit/storage.coffee

```diff
@@ -0,0 +1,65 @@
+kd = require 'kd'
+FlexSplit = require './index'
+
+
+module.exports = class FlexSplitStorage extends kd.Object
+
+
+  constructor: (options = {}, data) ->
+
+    super options, data
+
+    @storage    = {}
+    @viewCount  = 0
+    @eventCount = 0
+
+    if adapterClass = @getOption 'adapter'
```

Comment:
I thought the same while implementing but then it didn't look good :) we can change it to `adapterClass` for sure.